### PR TITLE
Stripe integration tutorial

### PR DESCRIPTION
**Pull request by anna-engle**

### CODE REVIEW

File: userguide/tutorials/stripe_plugin.adoc [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+
+For the purpose of this tutorial, enter the following:
+
+image:https://raw.githubusercontent.com/annabaker/killbill-docs/v3/userguide/assets/img/tutorials/stripe_integration_new_product.png[align=center]
+
+Click 'Save' and you should see the following:
+
+image:https://raw.githubusercontent.com/annabaker/killbill-docs/v3/userguide/assets/img/tutorials/stripe_integration_existing_plans.png[align=center]
+
+=== Step 5:  Create a Stripe account
```
Comment (sbrossie) [inline userguide/tutorials/stripe_plugin.adoc]: Should be `step 6` (and also increment subsequent ones)

File: userguide/tutorials/stripe_plugin.adoc [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+
+Next, in the left panel, navigate to Developers > API Keys and click on "Reveal test key token."  You will need these keys once we configure the Stripe plugin for Kill Bill.
+
+=== Step 6:  Install the Stripe Plugin
+
+In Kaui, hover over the plug icon at the top of the page and click on 'KPM (Kill Bill Plugin Manager).'  
+
+Select "Install New Plugin" in the upper-right corner.
+
+In the list of plugins, fnd "stripe" and click on the cloud button to install.
```
Comment (sbrossie) [inline userguide/tutorials/stripe_plugin.adoc]: typo: `fnd` -> `find`

File: userguide/tutorials/stripe_plugin.adoc [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+
+After a few moments, refresh the `+/kpm+` page.  You should see the stripe-plugin listed with a status of 'STOPPED.'  Click the play button to start it.  If you refresh again, you should find the stripe-plugin listed with a status of 'RUNNING.'
+
+image:https://raw.githubusercontent.com/annabaker/killbill-docs/v3/userguide/assets/img/tutorials/stripe_integration_plugin_running.png[align=center]
+
+=== Step 7:  Configure the Stripe Plugin with your API Keys
+
+In Kaui, at the top of the screen, hover over the gear icon and select 'TEnANTS.'
```
Comment (sbrossie) [inline userguide/tutorials/stripe_plugin.adoc]: typo: `TEnANTS ` -> `TENANTS `

File: userguide/tutorials/stripe_plugin.adoc [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+
+Click 'bob.'
+
+At the bottom of the screen, navigate to the tab labeled 'Plugin Config.'
+
+For Plugin name, select "stripe" from the drop-down menu.
+
+In the confiuration, enter the following and paste your publishable and secret keys you obtained from Stripe in Step 5.
```
Comment (sbrossie) [inline userguide/tutorials/stripe_plugin.adoc]: Will become Step 6

**Review by sbrossie — COMMENTED** [review]
Looking good! I pointed towards a few typos.

File: userguide/tutorials/stripe_plugin.adoc [hunk 5]
```diff
@@ -0,0 +1,230 @@
...
+Save the file, and run:
+
+[source,bash]
+----
+$ docker-compose up
+----
+
+The startup sequence lasts a couple of minutes.  You should see something like the following once it finishes:
+
+image:https://raw.githubusercontent.com/annabaker/killbill-docs/v3/userguide/assets/img/tutorials/stripe_integration_docker_startup.png[align=center]
```
Comment (pierre) [inline userguide/tutorials/stripe_plugin.adoc]: Can you update the links to point to our repo?

File: userguide/tutorials/stripe_plugin.adoc (hunk 4 above)
Comment (pierre) [inline userguide/tutorials/stripe_plugin.adoc]: Typo: `configuration`

File: userguide/tutorials/stripe_plugin.adoc [hunk 6]
```diff
@@ -0,0 +1,230 @@
...
+
+image:https://raw.githubusercontent.com/annabaker/killbill-docs/v3/userguide/assets/img/tutorials/stripe_integration_subscription_bundles.png[align=center]
+
+Navigating to the 'Payments' tab at the top of the page, you will also see that a payment was processed with an AUTH AMOUNT of $10.00.
+
+
+image:https://raw.githubusercontent.com/annabaker/killbill-docs/v3/userguide/assets/img/tutorials/stripe_integration_payments.png[align=center]
+
+
+=== Step 10:  Conclusion
```
Comment (pierre) [inline userguide/tutorials/stripe_plugin.adoc]: One idea would be to provide links for next steps:

* API reference (https://killbill.github.io/slate/)
* Production deployment (https://docs.killbill.io/latest/aws.html)
* Community forum for help (https://groups.google.com/forum/#!forum/killbilling-users)

File: userguide/tutorials/stripe_plugin.adoc (hunk 5 above)
Comment (anna-engle) [reply userguide/tutorials/stripe_plugin.adoc]: Sure, they will be broken though until this PR is merged.  Next time, I will create a separate PR for screenshots.

Comment (pierre) [reply userguide/tutorials/stripe_plugin.adoc]: 👍

**Comment by pierre** [comment]
Thank you!

**Comment by pierre** [comment]
FYI I've updated the blog post (https://killbill.io/blog/having-fun-with-docker-stripe-and-kill-bill-v2/) and https://docs.killbill.io/

**Pull request merged**